refactor(ast_tools): shorten code

overlookmotel opened the pull request:
#5904 reduced the generated code for `Visit` trait. Shorten the codegen a little after that change.

overlookmotel commented:
* **#5915** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/5915?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

camchenry reviewed (approved):
Nice, this seems like a good follow up to me

overlookmotel commented:
Thanks for reviewing @camchenry!

overlookmotel committed:
refactor(ast_tools): shorten code (#5915)

#5904 reduced the generated code for `Visit` trait. Shorten the codegen a little after that change.
